src/features/events/components/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+import NextLink from 'next/link';
+import { Settings } from '@mui/icons-material';
+import {
+  Avatar,
+  Box,
+  ClickAwayListener,
+  Divider,
+  Link,
+  Paper,
+  Popper,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { FC, useState } from 'react';
+
+import EventDataModel from 'features/events/models/EventDataModel';
+import messageIds from 'features/events/l10n/messageIds';
+import { personResource } from 'features/profile/api/people';
+import theme from 'theme';
+import { useMessages } from 'core/i18n';
+import ZUICard from 'zui/ZUICard';
+import ZUINumberChip from 'zui/ZUINumberChip';
+import ZUIPersonLink from 'zui/ZUIPersonLink';
+
+type EventParticipantsCardProps = {
+  campId: string;
+  model: EventDataModel;
+};
+
+const EventParticipantsCard: FC<EventParticipantsCardProps> = ({
+  model,
+  campId,
+}) => {
+  const eventData = model.getData().data;
+  const messages = useMessages(messageIds);
+  const reqParticipants = eventData?.num_participants_required ?? 0;
+  const availParticipants = eventData?.num_participants_available ?? 0;
+
+  const [newReqParticipants, setNewReqParticipants] = useState<number | null>(
+    reqParticipants
+  );
+  const [anchorEl, setAnchorEl] = useState<
+    null | (EventTarget & SVGSVGElement)
+  >(null);
+
+  if (!eventData) {
+    return null;
+  }
+
+  let contact = null;
+  if (eventData.contact) {
+    contact = personResource(
+      eventData.organization.id.toString(),
+      eventData.contact.id.toString()
+    ).useQuery().data;
```

Comment:
This is using an older solution that we are moving away from. It's still ok to use it when no alternative exists, but it jumped out at me. I see why you did it though, but with my suggestion to not use `ZUIPersonLink`, hopefully we can let go of this again?